build/config/PublishData.json

```diff
@@ -1,18 +1,18 @@
 {
     "branches": {
-        "dev15.5.x": {
+        "master": {
```

Comment:
This should be master-vs-deps, as that's where we'll be publishing packages.

Reply:
(or alternately: if you *do* want to publish out of master, then the channels needs some way to distinguish)